server/grpc/server.go

```diff
@@ -92,7 +99,21 @@ func (s *Server) register() {
 	protobuf_api.RegisterOwnershipServer(s.instance, api.NewOwnershipServer(s.mc))
 	protobuf_api.RegisterRunnerServer(s.instance, api.NewRunnerServer(s.mc, s.b))
 
-	runner.RegisterRunnerServer(s.instance, runner.NewServer(s.mc.RPC, s.ep))
+	// decode public keys
+	// TODO: this is not the best place. it should be done directly in config
+	pks := make([]crypto.PubKey, 0)
+	for _, pkS := range s.authorizedPubKeys {
```

Comment:
It would be nice to have the same format that we use to sign transactions.

eg:
```js
{
      signature: Buffer.from(signature).toString('base64'),
      pub_key: {
        type: "tendermint/PubKeySecp256k1",
        value: Buffer.from(pubKeyByte).toString('base64')
      }
}
```

This way the CLI or any other will not have to deal with another prefix. What do you think?

Reply:
Cosmos already has a builtin way to encode and decoded pubkeys in a similar way than addresses: prefix (`mesgtestpub`) + bech32.

What you are showing is the amino-json-encoded of a `tendermint/PubKeySecp256k1` struct.
I think it makes sense to keep the bech32 encoding in yaml file where users should be able to edit it.

Moreover, the cosmos cli already display the pubkey in bech32. I think we should simply do the same.
```
➜  engine git:(feature/e2e-docker-cli) ./bin/mesg-cli keys list
- name: e2e
  type: local
  address: mesgtest1797drk09dqw60wd9ffvj7hag68cqxhnfex6q0x
  pubkey: mesgtestpub1addwnpepqg0ujk8vcwq86z8466rznfvfz5rk992wsy0qtczfedgsy3x90mrzk476x05
  mnemonic: ""
  threshold: 0
  pubkeys: []
- name: engine
  type: local
  address: mesgtest1sfjx3xmrnrllhc86jhfzdpqpckn4s69a5tpr4f
  pubkey: mesgtestpub1addwnpepqwl7axwafcqnc9z92eyrl835w6j54r8806x3lnnqedydt47j6papjmn0439
  mnemonic: ""
  threshold: 0
  pubkeys: []
```